market/src/main/java/university/market/base/exception/GlobalExceptionHandler.java

```diff
@@ -11,13 +11,8 @@
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
-    @ExceptionHandler(MemberException.class)
-    public ResponseEntity<ErrorResponse> handleMemberException(MemberException ex) {
-        return createErrorResponse(ex.exceptionType().httpStatus(), ex.exceptionType().errorMessage(), ex.exceptionType().errorCode());
-    }
-
-    @ExceptionHandler(EmailException.class)
-    public ResponseEntity<ErrorResponse> handleEmailException(EmailException ex) {
+    @ExceptionHandler({MemberException.class, EmailException.class})
```

Comment:
```suggestion
    @ExceptionHandler(BaseException.class)
```
그냥 BaseException 만 처리하는 것은 어떨까요?

Reply:
네 그렇게 변경하였습니다!! 감사합니다